Remarkable-Donkeys/bangazonWebApp
# Jf messing around

**The Name of this branch is jf-messing-around**


* **Please check if the PR fulfills these requirements**


* **What kind of change does this PR introduce?** (Bug Fix, Feature, Document Update...)
- properly seeds category table if the category table has 0 rows
- properly seeds product table if the product table has 0 rows



* **What Ticket is associated with this PR**
None


* **What is the new behavior?**  (if this is a feature change)
- seeds category table
- seeds product table

* **List all core functionality**


* **Other information / Steps to Test**:
1. run the following in the terminal
```
git fetch --all
git checkout jf-messing-around
```

2. run the application

3. register a user with an email of "jdoe@email.com"

4. logout of that user

5. register a user with an email of "jsmith@email.com" 

6. close the application

7. re-run the application

8. close the application

9. check the category table in the sql server explorer sidebar and the table should be populated

10. check the product table in the sql server explorer sidebar and the table should be populated  

***At least two people need to approve this pull request before merged to Development.***
